### DISCUSSION TITLE
Is it possible to start WSA without opening any apps?

### DISCUSSION THREAD

**Discussion by userandaname** [microsoft/WSA#207] [General]
Hi, I was wondering if it's possible to start WSA without opening any apps and know when WSA has finished starting?

Is this possible?

**Comment by infinitepower18** [comment]
wsa://system

Also, I'm working on an [utility](https://github.com/infinitepower18/WSA-SystemControl) to monitor WSA status from taskbar including the option to start WSA exactly the way you mentioned.

**Reply by userandaname** [reply]
Wow, thank you so much! I'll keep your utility in mind later on!
